index.html

```diff
@@ -0,0 +1,72 @@
+<!-- ### Content and Technical Requirements
+1. Article element containing your bio
+1. Article element with links to resources or sites you like
+1. Article element with two sections for future projects (placeholders)
+1. Footer element with email and professional social media links; Github, twitter, LinkedIn, etc.
+1. Semantic mark-up for all major elements.
+1. 1. Validate your html page with W3 validator: https://validator.w3.org/
+1. Appropriate folder structure: images, css.
+ --><!DOCTYPE html>
+<html>
+<head>
+	<title>Trent Hand:Full-Stack Developer</title>
+	<link rel="stylesheet" type="text/css" href="css/quiz.css">
+	<link href="//netdna.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.css" rel="stylesheet">
+</head>
+<body>
+	
+	<div class="wrapper">  
```

Comment:
Be very careful about leaving trailing whitespace in your code. It will inevitably lead to merge conflicts. You can see whitespace by adding a plugin called Trailing Spaces to your sublime, or you can add this line to your User Preferences and stop them out right. "trim_trailing_white_space_on_save": true